Aliucord/src/main/java/com/aliucord/PluginManager.java

```diff
@@ -202,4 +204,17 @@ public static boolean isPluginEnabled(String name) {
     public static boolean isPluginEnabled(Plugin plugin) {
         return isPluginEnabled(MapUtils.getMapKey(plugins, plugin));
     }
+
+    /**
+     * Prompts the user to restart Aliucord because of a plugin that requires a restart
+     */
+    public static void promptRestart() {
```

Comment:
Make this part of Utils perhaps? Maybe even with a custom message that defaults to the one you specified here? Also can't you just use Utils.appActivity as view